_sass/_header.scss

```diff
@@ -5,6 +5,7 @@
 .navbar-custom {
   margin-bottom: 0;
   font-family: $montserrat;
+  background-color: #fff;
```

Comment:
Is this needed?

Reply:
Yeah – it’s unrelated but it fixes the header having no background on mobile.